test/evergreen.yml

```diff
@@ -2148,15 +2146,6 @@ tasks:
       - func: "fetch artifacts"
       - func: "unit test"
 
-  - name: unit-test-macos
-    tags: ["python"]
-    depends_on:
-    - name: compile
-    commands:
-      - func: "fetch artifacts"
-      - func: "python config check"
```

Comment:
I see this function was only called here, should we get rid of the function as well?

Reply:
I think we can remove

Reply:
Don't see the point of python_version_check.py now. Should we remove it as well?

Reply:
removed